docs/_config.yml

```diff
@@ -11,7 +11,11 @@ github_username: auth0
 github_repo: auth0-cli
 
 # Build settings
-theme: minima
+remote_theme: just-the-docs/just-the-docs@v0.4.0.rc3
 
-header_pages:
-  -
+search_enabled: true
+favicon_ico: "/assets/images/favicon.png"
```

Comment:
**Question:** Do we really need the intermediate `images` directory for a single favicon? I'd even question if the `assets` directory makes sense.

Reply:
No real reason I did this other than organisation, happy to move back into the toplevel directory if you want

Reply:
It is ok. So long as there's some rationale.